canonical/juju-dashboard
# WD-8398 - View secret contents

## Done

- Fetch and display secret contents in a modal.

## QA

- Go to the secrets tab.
- Click the eye next to a secret.
- Choose a revision and click "View"
- The secret contents should be displayed.

## Details

https://warthogs.atlassian.net/browse/WD-8398
